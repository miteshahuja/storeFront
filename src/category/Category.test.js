import renderer from 'react-test-renderer';
import React from 'react';
import Enzyme, { shallow } from 'enzyme'
//import Adapter from 'enzyme-adapter-react-16'
import ReactDOM from 'react-dom';
import CategoryTest from './Category';

import { connect } from "react-redux";



it('render correctly Category component', () => {
  const Category = renderer.create(connect(<CategoryTest />)).toJSON();
  expect(Category).toMatchSnapshot();
});