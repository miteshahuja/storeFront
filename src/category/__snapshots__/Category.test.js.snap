// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Category component 1`] = `null`;
